content/modals/US/product_list.json

```
@@ -6,10 +6,10 @@
         "headline": "Pay later with flexible options",
         "disclosure": "Terms apply for each option. Offer availability may depend on consumer & merchant eligibility.",
         "instructions": {
-            "top": "Choose an option for more details.",
-            "bottom": "Select <strong>PayPal</strong> at checkout to <strong>Pay Later</strong>."
+            "payLater": "Choose a way to pay later",
+            "credit": "Or shop with a reusable credit line."
         },
-        "tiles": [
+        "tiles1": [
```

```suggestion
        "tiles1": [
```
Can this array be removed? It looks like `tiles` right below it is more up to date with this change.